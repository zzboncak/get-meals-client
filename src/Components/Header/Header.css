.header-section__links {
        width: 100%;
        background: #f18752;
        padding: 8px 16px;
}

/* Link {
    text-decoration: none;
}

span {
    text-decoration: none;
    color: blue;
} */

.header-section__links > div {
    display: inline-block;
    vertical-align: middle;
}

.header-section__link-homepage {
    color: #f37335;
    text-decoration: none;
    font-weight: bold;
    font-size: 22px;
}

.header-section__nav {
    display: inline-block;
    text-align: right;
    width: calc(100% - 74px);
    vertical-align: middle;
}

.header-section__links {
    margin: 0;
    padding: 0;
    list-style: none;
}

.header-section__link {
    display: inline-block;
    margin: 0 16px;
}

.header-section__link Link {
    text-decoration: none;
    color: #f37335;
    font-weight: bold;
    padding: 3px 0;
}

.header-section__link Link:hover,
.header-section__link Link:active {
    color: white;
    border-bottom: 5px solid white;
}




/* .header-section__links {
    border-bottom: solid 1px #f37335;
    background-color: #f37335;
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    margin-top: 20px;
    min-height: 50px;
    font-size: 16px;
}

.header-section__link-homepage {
    display: flex;
    flex-direction: row;
    justify-content: left;
    align-items: center;
    margin-top: 20px;
    min-height: 50px;
    font-size: 16px;
}

.header-section__link {
    display: flex;
    flex-direction: row;
    justify-content: left; 
    align-items: center;
    margin-top: 20px;
    min-height: 50px;
    font-size: 16px;
}

.header-mobile {
    background-color: darkgray;
}

.header-fullscreen {
    display: none;
}

img.header-mobile {
    width: 40px;
    height: 40px;
}

ul.header-mobile {
    margin-top: 0;
    margin-bottom: 20px;
    list-style: none;
    text-align: right;
}

breakpoint for tablets
@media (min-width: 728px) {
    
}

breakpoint for screens
@media (min-width: 1024px) {
    .header-container {
        justify-content: space-around;
        margin: 20px 0;
    }

    .header-fullscreen {
        display: block;
    }

    .header-mobile {
        display: none;
    }
} */
