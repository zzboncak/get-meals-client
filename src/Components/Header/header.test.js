import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './header';
import { BrowserRouter } from 'react-router-dom';

describe(`Header component`, () => {
    const props = {
        className: 'test-class-name'
    }

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the Header component as expected', () => {
        const tree = renderer
          .create(
            <BrowserRouter>
                <Header />
            </BrowserRouter>)
          .toJSON();
        expect(tree).toMatchSnapshot();  
    });

})
