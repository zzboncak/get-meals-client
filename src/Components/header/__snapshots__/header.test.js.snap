// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the Header component as expected 1`] = `
<div
  className="header-container"
>
  <section>
    GetMealz
  </section>
  <section>
    Home Page
  </section>
  <section>
    About Us
  </section>
  <section>
    Add Location
  </section>
</div>
`;
